app/src/main/java/org/p2p/wallet/newsend/SendModule.kt

```diff
@@ -90,5 +99,17 @@ object SendModule : InjectionModule {
         factoryOf(::FeeRelayerViaLinkInteractor)
         factoryOf(::SendViaLinkInteractor)
         factoryOf(::UserSendLinksDatabaseRepository) bind UserSendLinksLocalRepository::class
+
+        factoryOf(::SendServiceInteractor)
+        singleOf(::SendServiceRemoteRepository)
```

Comment:
you sure you want to make remote repo as a singleton? and without interface?

Reply:
agree, there has `SendServiceRepository` interface and stateless `SendServiceRemoteRepository`, so it easily can be a `factoryOf(::SendServiceRemoteRepository) bind SendServiceRepository::class`